lib/reverse_proxy_plug/http_client/adapters/req.ex

```diff
@@ -94,13 +94,20 @@ if Code.ensure_loaded?(Req) do
       end
 
       defp async_request(req, parent) do
+        connect_timeout =
+          req.options
+          |> Keyword.get(:connect_options, [])
+          |> Keyword.get(:timeout, 30_000)
+
+        recv_timeout = Keyword.get(req.options, :recv_timeout, 15_000)
```

Comment:
looks like the option for req is `:receive_timeout` not `:recv_timeout`

```suggestion
        recv_timeout = Keyword.get(req.options, :receive_timeout, 15_000)
```